pulp_smash/cli.py

```diff
@@ -359,6 +359,22 @@ def _restart_systemd(client, services):
         cmd = ('systemctl', 'restart') + tuple(services)
         return (client.run(cmd, sudo=True),)
 
+    @staticmethod
+    def _is_active_sysv(client, services):
+        with contextlib.suppress(exceptions.CalledProcessError):
+            return tuple((
+                client.run(('service', service, 'status'), sudo=True)
+                for service in services
+            ))
+        return False
+
+    @staticmethod
+    def _is_active_systemd(client, services):
+        with contextlib.suppress(exceptions.CalledProcessError):
+            cmd = ('systemctl', 'is-active') + tuple(services)
+            return (client.run(cmd, sudo=True),)
```

Comment:
This will return exit code `0` if any of the services are active examples 

```bash
$ systemctl is-active pulp_worker@*
$ echo $?
0  # if any of @{1..m} is running

$systemctl is-active pulp_worker@1 pulp_worker@2 foobar
$ echo $?
0  # if any of above is running

$ systemctl is-active foobar
$ echo $?
3  # specific service error because foobar is not active, is this case we suppress the error and return False
```